Misc/NEWS.d/next/Library/2018-05-15-15-03-48.bpo-28612.E9dz39.rst

```diff
@@ -0,0 +1,2 @@
+Added support for optional Site Map extension to urllib robotparser. Patch
+by Lady Red
```

Comment:
Please end the sentence with a period. In addition, since it was based off another person's patch, it would be good to also mention `Based on patch by Peter Wirtz.`